core/src/main/java/greencity/controller/FileServiceController.java

```diff
@@ -45,6 +47,7 @@ public FileServiceController(FileService fileService) {
         @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = EcoNews.class),
         @ApiResponse(code = 303, message = HttpStatuses.SEE_OTHER),
         @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+        @ApiResponse(code = 500, message = HttpStatuses.INTERNAL_SERVER_ERROR),
     })
     @PostMapping("/image")
     public ResponseEntity<String> uploadImage(@RequestParam("Image to save") @NotEmpty MultipartFile multipartFile,
```

Comment:
FIle for sure should be send in body, so it should be @RequestBody or @RequestPart annotation. It looks like you haven't test it - because you won't be able to send file in URL

Reply:
Also bad naming - use cammelCase